docker/entry

```diff
@@ -28,10 +28,15 @@ subprocess.call(["useradd", "build",
 
 # Become the 'build' user.  If no command argument was provided,
 # start an interactive shell.
-su_cmd = ["su", "-", "build"]
+sudo_cmd = ["sudo", "-u", "build", "-i"]
+
+SSH_AUTH_SOCK = os.getenv("SSH_AUTH_SOCK")
```

Comment:
I find the `is not None` ugly so I'd write this as

    if 'SSH_AUTH_SOCK' in os.environ:
        sudo_cmd += ["SSH_AUTH_SOCK="+os.environ['SSH_AUTH_SOCK']]